style.css

```diff
@@ -402,3 +421,75 @@ body {
   flex-direction: column;
   text-align: center;
 }
+
+/* CONTACT PART */
+.footer {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  width: 100%;
+  height: auto;
+}
+
+.footer-content {
+  align-items: center;
+  justify-content: center;
+}
+
+.footer-text {
+  padding-top: 3rem;
+}
+
+#form {
+  padding-left: 1.5rem;
+}
+
+#form #label {
+  color: #979493;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+#form input {
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  width: 20rem;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+  padding: 5px;
+}
+
+#textarea {
+  margin-top: 2rem;
+  background: #fbf8f7;
+}
+
+.btn {
+  background: #ff6b00;
+  border: none;
+  color: #fff;
+  margin: 2rem 8rem;
+  padding: 12px 3px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+.btn:hover {
+  background: rgba(255, 255, 255, 0.24);
+  color: #ff6b00;
+  border: 2px solid #ff6b00;
+}
+
+.down-boarder {
+  padding: 0.5rem 0 0 8rem;
+}
```

Comment:
- [ ] Your design is still not quite similar to the Figma design. Kindly try to fix the widths of the inputs, the size of the button, etc.
Your Design:
![image](https://user-images.githubusercontent.com/61919617/172959804-c4076841-5554-4bc2-8e38-054cd0aeec2d.png)